start-a-local-cluster.md

```diff
@@ -70,7 +68,9 @@ Data is stored on-disk and client/server communication is secure. Setup involves
     # To exit: CTRL + D.
     ```
 
-4. [Run some queries](/basic-sql-commands.html).
+5. [Run some queries](/basic-sql-commands.html).
+
+6. Check out the Admin UI by pointing your browser to `<your local host>:26257`. You can find your local host by running `hostname` in your shell.    
```

Comment:
In secure mode, you must specify `https://<your local host>:26257`. This will also give a security warning that must be clicked past.